boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -332,3 +333,27 @@ opm-build-push: docker-push
 .PHONY: ensure-fips
 ensure-fips:
 	${CONVENTION_DIR}/configure-fips.sh
+
+.PHONY: migrate-tools
+migrate-tools:
+	# Installing controller-gen v0.8.0
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_GEN_VERSION}
+	mv ${GOBIN}/controller-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+	# Installing openapi-gen v0.23.0
+	go install k8s.io/code-generator/cmd/openapi-gen@${OPENAPI_GEN_VERSION}
+	mv ${GOBIN}/openapi-gen ${GOBIN}/controller-gen-${CONTROLLER_GEN_VERSION}
+
+.PHONY: migrate-files
+# You will need to export the forked/cloned operator repository directory as REPO to make this work.
+# Example: export REPO=~/Projects/My-Operator-Fork
```

Comment:
Suggest
- Adding a check to make sure this variable is defined. Otherwise `rsync` will end up trying to copy stuff from under `/`, which probably won't work, but could be pretty bad if it does. See L1-16 of this file for examples of such checks.
- Naming the variable something more descriptive, like `$(OLD_SDK_REPO_DIR)`.